### ISSUE TITLE
Did not get the correct type of variable (calling pandas.read_csv)

### ISSUE THREAD

**Issue by huyiqun** [microsoft/pyright#1422]
**Describe the bug**
When using `read_csv` function from `pandas`, setting `iterator` to `False` (default behavior) should return a `DataFrame` object, which should have all `DataFrame` methods. However, pyright seems to decide that it returns the `TextFileReader` object, which should only be the case when `iterator` is set to `True` or `chunksize` is not `None`.

Since all my remaining code tries to analyze the data assuming the object is `DataFrame`, pyright shows that every line has errors.

**To Reproduce**
```
import pandas as pd
df = pd.read_csv("any.csv")
print(df.head)
```

**Expected behavior**
`df` should be a `DataFrame` object and accessing all `DataFrame` methods should not report as errors.

**Screenshots or Code**
![image](https://user-images.githubusercontent.com/10877536/105803365-565ce500-5f6b-11eb-8916-211740040b6a.png)
![image](https://user-images.githubusercontent.com/10877536/105803331-3b8a7080-5f6b-11eb-9564-5295f1138e6e.png)

**VS Code extension or command-line**
Using coc-pyright extension (1.1.104)

**Comment by erictraut** [comment]
Pyright is doing the right thing in this case. Its type analysis is based on the type information provided via type stubs. The pandas type stubs don't ship with Pyright, so you must have installed them from some other source. If you think the pandas type stubs are in error, please file bugs against the associated repo.

**Comment by huyiqun** [comment]
Did not know about this, thanks for explaining the details!

**Comment by jvivian-atreca** [comment]
@huyiqun — how did you fix this? I installed `pandas-stubs` but that didn't fix the issue for me.

**Comment by mochnaufals** [comment]
> @huyiqun — how did you fix this? I installed `pandas-stubs` but that didn't fix the issue for me.

I did install pandas-stubs, and it is actually worked for me.

**Comment by nyngwang** [comment]
@mochnaufals No need to install `pandas-stubs`. With these two options work for me:

```lua
        typeCheckingMode = 'off',
        useLibraryCodeForTypes = true,
```
